yet-another-docker-plugin/src/main/java/com/github/kostyasha/yad/launcher/DockerComputerJNLPLauncher.java

```diff
@@ -55,13 +55,23 @@
 
     /**
      * Configured from UI
+     * @deprecated because properties moved to fields
      */
-    protected JNLPLauncher jnlpLauncher = new JNLPLauncher();
+    @Deprecated
+    protected transient JNLPLauncher jnlpLauncher;
 
     protected long launchTimeout = DEFAULT_TIMEOUT; //seconds
 
     protected String user = "jenkins";
 
+    protected String jvmOpts;
```

Comment:
Yes, you can't really avoid nulls, because as soon as xstream will load it from old config, it will be null. But i usually use to indicate expected defaults. Real null avoid should happen in getter.